src/ucp/proto/proto_reconfig.c

```diff
@@ -69,24 +69,25 @@ static ucs_status_t ucp_proto_reconfig_progress(uct_pending_req_t *self)
 static ucs_status_t
 ucp_proto_reconfig_init(const ucp_proto_init_params_t *init_params)
 {
+    ucp_proto_perf_range_t *perf_range = &init_params->caps->ranges[0];
     ucp_proto_perf_type_t perf_type;
 
     /* Default reconfiguration protocol is a fallback for any case protocol
      * selection is unsuccessful. The protocol keeps queuing requests until they
      * can be executed.
      */
-    *init_params->priv_size                 = 0;
-    init_params->caps->cfg_thresh           = UCS_MEMUNITS_INF;
-    init_params->caps->cfg_priority         = 0;
-    init_params->caps->min_length           = 0;
-    init_params->caps->num_ranges           = 1;
-    init_params->caps->ranges[0].max_length = SIZE_MAX;
-    init_params->caps->ranges[0].node       = NULL;
+    *init_params->priv_size         = 0;
+    init_params->caps->cfg_thresh   = UCS_MEMUNITS_INF;
+    init_params->caps->cfg_priority = 0;
+    init_params->caps->min_length   = 0;
+    init_params->caps->num_ranges   = 1;
```

Comment:
can we unite initializing `caps` here with `ucp_proto_select_caps_cleanup`? the only difference between them is `num_ranges` - can be passed as parameter to common function